src/bowling_SC.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+class bowlingGame {
+
+  constructor() {
+    this.rolls = [[], [], [], [], [], [], [], [], [], []];
+    this.runningScore = 0;
+    this.frameIndex = 1;
+    this.rollIndex = 0;
+  };
+
+  firstRoll(points) {
+    this.rolls[this.frameIndex -1].push(points);
+    this.runningScore += points;
+    if (points === 10) {
+      this.frameIndex++
+    };
+  };
+
+  secondRoll(points) {
+    this.rolls[this.frameIndex -1].push(points);
+    this.runningScore += points;
+    this.frameIndex++
+  };
+
+  spareCalc() {
```

Comment:
It looks like this function is only called within a test. Is this a function that a user would be expected to remember to call themselves or could we add a call to this function somewhere in the code?